server/src/main/java/org/opensearch/cluster/coordination/FollowersChecker.java

```diff
@@ -395,13 +396,13 @@ public void handleException(TransportException exp) {
 
                         final String reason;
                         if (exp instanceof ConnectTransportException || exp.getCause() instanceof ConnectTransportException) {
-                            logger.debug(() -> new ParameterizedMessage("{} disconnected", FollowerChecker.this), exp);
+                            logger.info(() -> new ParameterizedMessage("{} disconnected", FollowerChecker.this), exp);
                             reason = "disconnected";
                         } else if (exp.getCause() instanceof NodeHealthCheckFailureException) {
-                            logger.debug(() -> new ParameterizedMessage("{} health check failed", FollowerChecker.this), exp);
+                            logger.info(() -> new ParameterizedMessage("{} health check failed", FollowerChecker.this), exp);
                             reason = "health check failed";
                         } else if (failureCountSinceLastSuccess >= followerCheckRetryCount) {
-                            logger.debug(() -> new ParameterizedMessage("{} failed too many times", FollowerChecker.this), exp);
+                            logger.info(() -> new ParameterizedMessage("{} failed too many times", FollowerChecker.this), exp);
                             reason = "followers check retry count exceeded";
                         } else {
                             logger.debug(() -> new ParameterizedMessage("{} failed, retrying", FollowerChecker.this), exp);
```

Comment:
Wondering if retries should be info as well, since it would be helpful to know if single attempts failed too

Reply:
The purpose was to troubleshoot reasons node-left if that failure occurs; Single attempts failing seems like additional information more suited for the debug level - especially in a production system not opted into debugging. My concern was that it might unnecessarily crowd the output - especially in a system with large number of followers since this would all print on the leader node.

Reply:
Its possible for node-left due `followers check retry count exceeded` to have different reasons per attempt like timeout, CSRejectedException etc. The above logging will point the last attempt failure reason as the culprit, which might be leading. So either we could aggregate those reasons or log all. Thoughts?

Reply:
Yes, I see your point. In that case, it makes more sense to aggregate reasons and print all of them on followers check retry count exceeded.  Would that work?